pkg/utils/net-attach-def.go

```diff
@@ -78,24 +78,22 @@ func setPodNetworkStatus(client kubernetes.Interface, pod *corev1.Pod, networkst
 	}
 
 	coreClient := client.CoreV1()
-
-	pod.Annotations[v1.NetworkStatusAnnot] = networkstatus
-	pod.Annotations[v1.OldNetworkStatusAnnot] = networkstatus
-	pod = pod.DeepCopy()
 	var err error
+	name := pod.Name
```

Comment:
Please use the commit message to describe what exactly is broken now and how is your change fixing it.